cmd/swarmctl/secrets/inspect.go

```diff
@@ -0,0 +1,57 @@
+package secrets
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	"github.com/docker/swarmkit/api"
+	"github.com/docker/swarmkit/cmd/swarmctl/common"
+	"github.com/spf13/cobra"
+)
+
+func printSecretSummary(secret *api.Secret) {
+	w := tabwriter.NewWriter(os.Stdout, 8, 8, 8, ' ', 0)
+	defer w.Flush()
+
+	common.FprintfIfNotEmpty(w, "ID\t: %s\n", secret.ID)
+	common.FprintfIfNotEmpty(w, "Name\t: %s\n", secret.Spec.Annotations.Name)
+	if len(secret.Spec.Annotations.Labels) > 0 {
+		fmt.Fprintln(w, "Labels\t")
+		for k, v := range secret.Spec.Annotations.Labels {
+			fmt.Fprintf(w, "  %s\t: %s\n", k, v)
+		}
+	}
+	common.FprintfIfNotEmpty(w, "Digest\t: %s\n", secret.Digest)
+	common.FprintfIfNotEmpty(w, "Size\t: %d\n", secret.SecretSize)
+
+	created := time.Unix(int64(secret.Meta.CreatedAt.Seconds), int64(secret.Meta.CreatedAt.Nanos))
```

Comment:
`s.Meta.CreatedAt` seems to be the wrong type (I get `type *"github.com/docker/swarmkit/api/timestamp".Timestamp) as type *"github.com/golang/protobuf/ptypes/timestamp".Timestamp in argument to ptypes.Timestamp` when I tried to do it previously), so none of the `ptypes` functions work with it.  Am I missing something in the protocols, or some kind of translation, etc. that I'm supposed to do such that it's the right type?

I can convert to a `ptypes` type instead of a `time.Time` type to use it with that package instead?